pkg/apis/storage/types.go

```diff
@@ -293,8 +293,44 @@ type CSIDriverSpec struct {
 	// "csi.storage.k8s.io/pod.uid": string(pod.UID)
 	// +optional
 	PodInfoOnMount *bool
+
+	// VolumeModes defines what kind of volumes this CSI volume driver supports.
+	// The default if the list is empty is "Persistent", which is the usage
```

Comment:
The default mode if the list is empty is "Persistent"

Reply:
Yep, copied from the other types.go.